src/pages/workspace/accounting/netsuite/export/NetSuiteExportConfigurationPage.tsx

```diff
@@ -0,0 +1,258 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import ConnectionLayout from '@components/ConnectionLayout';
+import type {MenuItemProps} from '@components/MenuItem';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import type {OfflineWithFeedbackProps} from '@components/OfflineWithFeedback';
+import useLocalize from '@hooks/useLocalize';
+import usePermissions from '@hooks/usePermissions';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import {canUseProvincialTaxNetSuite, canUseTaxNetSuite} from '@libs/PolicyUtils';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import type {ToggleSettingOptionRowProps} from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type {Errors} from '@src/types/onyx/OnyxCommon';
+
+type MenuItem = MenuItemProps & {
+    pendingAction?: OfflineWithFeedbackProps['pendingAction'];
+
+    shouldHide?: boolean;
+
+    /** Any error message to show */
+    errors?: Errors;
+
+    /** Callback to close the error messages */
+    onCloseError?: () => void;
+
+    type: 'menuitem';
+};
+
+type DividerLineItem = {
+    type: 'divider';
```

Comment:
Do you think we'll need to use for Import/Advanced? Then should we move to a separate type?